### PR TITLE
Add zen 5.0.2

### PR DESCRIPTION
**Pull request by cronicc**

### CODE REVIEW

File: dists/bookworm/main/binary-amd64/Packages [hunk 1]
```diff
@@ -1,3 +1,19 @@
+Package: zen
+Version: 5.0.2
+Architecture: amd64
+Maintainer: Zen Blockchain Foundation <info@horizen.io>
```
Comment (ysibirski) [inline dists/bookworm/main/binary-amd64/Packages:4]: Cronic are we planning to change ZBF to Horizen Foundation at some point or not really?

**Review by ysibirski — APPROVED** [review]
LGTM + one small comment/question :)

File: dists/bookworm/main/binary-amd64/Packages (hunk 1 above)
Comment (cronicc) [reply dists/bookworm/main/binary-amd64/Packages:4]: It would have to be changed on zen repo. I'll create a ticket.

**Review by lander86 — APPROVED** [review]
LGTM

**Pull request merged**